pattern_import_export/models/base.py

```diff
@@ -98,30 +98,34 @@ def _load_records_create(self, values):
         return super()._load_records_create(copy.deepcopy(values))
 
     def _pattern_format2json(self, row):
```

Comment:
What problem did you solve?

Reply:
When the number of records was greater than 9, the sorting ordered columns by the first digit, that is: 
instead of having 1, 2, 3...9, 10, 11...
we had 1, 10, 11... 2, 20, 21... 
Thus causing issues when importing